_notifications-plugin/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Notifications
+nav_order: 1
+has_children: false
+redirect_from:
+  - /notifications-plugin/
+---
+
+# Notifications
+
+The notifications plugin provides a central location for all of your notifications from OpenSearch plugins. Using the plugin, you can configure which communication service you want to use as well as see relevant statistics and troubleshooting information. Currently, the plugin supports sending notifications from the Alerting and Index State Management (ISM) plugins.
```

Comment:
Global note: In all files, please capitalize "Notifications" in "Notifications plugin". Same for "Alerting plugin".